openstreetmap-carto.lua

```diff
@@ -0,0 +1,447 @@
+      {'historic', 'citywalls'},
+      {'waterway', 'canal'},
+      {'waterway', 'derelict_canal'},
+      {'waterway', 'ditch'},
+      {'waterway', 'drain'},
+      {'waterway', 'river'},
+      {'waterway', 'stream'},
+      {'waterway', 'wadi'},
+      {'waterway', 'weir'},
+      {'power', 'line'},
+      {'power', 'minor_line'}
+   }
+
+-- The following keys will be deleted
+delete_tags = {
+  'note',
+  'note:.*',
+  'source',
+  'source_ref',
+  'source:.*',
+  'attribution',
+  'comment',
+  'fixme',
+  -- Tags generally dropped by editors, not otherwise covered
+  'created_by',
+  'odbl',
+  'odbl:note',
+  -- Lots of import tags
+  -- EUROSHA (Various countries)
+  'project:eurosha_2012',
+  -- Corine (CLC) (Europe)
+  'CLC:.*',
+
+  -- UrbIS (Brussels, BE)
+  'ref:UrbIS',
+
+  -- Geobase (CA)
+  'geobase:.*',
+  -- NHN (CA)
+  'accuracy:meters',
+  'sub_sea:type',
+  'waterway:type',
+  -- CanVec (CA)
+  'canvec:.*',
+  -- StatsCan (CA)
+  'statscan:rbuid',
+  -- Geobase (CA)
+  'geobase:.*',
+
+  -- RUIAN (CZ)
+  'ref:ruian:addr',
+  'ref:ruian',
+  'building:ruian:type',
+  -- DIBAVOD (CZ)
+  'dibavod:id',
+  -- UIR-ADR (CZ)
+  'uir_adr:ADRESA_KOD',
+
+  -- osak (DK)
+  'osak:.*',
+  -- kms (DK)
+  'kms:.*',
+  -- GST (DK)
+  'gst:feat_id',
+
+  -- Maa-amet (EE)
+  'maaamet:ETAK',
+
+  -- ngbe (ES)
+  -- See also note:es and source:file above
+  'ngbe:.*',
+
+  -- FANTOIR (FR)
+  'ref:FR:FANTOIR',
+
+  -- Friuli Venezia Giulia (IT)
+  'it:fvg:.*',
+
+  -- KSJ2 (JA)
+  -- See also note:ja and source_ref above
+  'KSJ2:.*',
+  -- Yahoo/ALPS (JA)
+  'yh:.*',
+
+  -- 3dshapes (NL)
+  '3dshapes:ggmodelk',
+  -- AND (NL)
+  'AND_nosr_r',
+
+  -- OPPDATERIN (NO)
+  'OPPDATERIN',
+
+  -- LINZ (NZ)
+  'LINZ2OSM:.*',
+  'linz2osm:.*',
+  'LINZ:.*',
+
+  -- Various imports (PL)
+  'addr:city:simc',
+  'addr:street:sym_ul',
+  'building:usage:pl',
+  'building:use:pl',
+  -- WroclawGIS (PL)
+  'WroclawGIS:.*',
+  -- TERYT (PL)
+  'teryt:simc',
+
+  -- RABA (SK)
+  'raba:id',
+
+  -- Naptan (UK)
+  'naptan:.*',
+
+  -- TIGER (US)
+  'tiger:.*',
+  -- GNIS (US)
+  'gnis:.*',
+  -- DCGIS (Washington DC, US)
+  'dcgis:gis_id',
+  -- National Hydrography Dataset (US)
+  'NHD:.*',
+  'nhd:.*',
+  -- Building Identification Number (New York, US)
+  'nycdoitt:bin',
+  -- Chicago Building Inport (US)
+  'chicago:building_id',
+  -- Louisville, Kentucky/Building Outlines Import (US)
+  'lojic:bgnum',
+  -- MassGIS (Massachusetts, US)
+  'massgis:way_id',
+
+  -- mvdgis (Montevideo, UY)
+  'mvdgis:.*',
+
+  -- misc
+  'import',
+  'import_uuid',
+  'OBJTYPE',
+  'SK53_bulk:load'
+}
+
+
+-- Array used to specify z_order and osmcarto_z_order per key/value combination.
+-- The former is used for backwards compatibility and for uses that use a single
+-- database for multiple rendering styles.
+-- Each element has the form {key, value, z_order, osmcarto_z_order, is_road}.
+-- If is_road=1, the object will be added to planet_osm_roads.
```

Comment:
the syntax is `t[{foo="bar"}]="baz"` but won't work here because the table compairson checks for if they are the same object, not if the  contents of the table are the same